kubernetes/overlays/dev/prompt-proto-service/prompt-proto-service.yaml

```diff
@@ -39,3 +51,10 @@ spec:
                   secretKeyRef:
                     name: usdf-prompt-processing-creds
                     key: password
+            - name: AWS_SHARED_CREDENTIALS_FILE
+              valueFrom:
+                  secretKeyRef:
+                    name: s3-credentials
+                    key: s3_credentials_ini_file
```

Comment:
Is it possible to do something similar for the PostgreSQL credentials and remove `make_pgpass.py`? (Just asking if it's possible; I'd rather not get distracted with trying to rework the current system on [DM-36712](https://jira.lsstcorp.org/browse/DM-36712)).

Reply:
Yes.  We can do that with the pgpass file and remove the make_pgpass.py file.